apache-hbase-mixin/dashboards/apache-hbase-regionserver-overview.libsonnet

```diff
@@ -0,0 +1,1306 @@
+local g = (import 'grafana-builder/grafana.libsonnet');
+local grafana = (import 'grafonnet/grafana.libsonnet');
+local dashboard = grafana.dashboard;
+local template = grafana.template;
+local prometheus = grafana.prometheus;
+
+local dashboardUid = 'apache-hbase-regionserver-overview';
+
+local promDatasourceName = 'prometheus_datasource';
+
+local promDatasource = {
+  uid: '${%s}' % promDatasourceName,
+};
+
+local regionsPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'server_region_count{job=~"$job", hbase_cluster=~"$hbase_cluster", instance=~"$instance"}',
+      datasource=promDatasource,
+      legendFormat='{{instance}}',
+      format='time_series',
+    ),
+  ],
+  type: 'stat',
+  title: 'Regions',
+  description: 'The number of regions hosted by the RegionServer.',
```

Comment:
In descriptions and panel titles I think we should use the term as "Region Server" instead of "RegionSever" since these are display names and they should not be formatted like variables IMO